docs/topics/mta-web-archetypes.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+
+[id="mta-archetypes_{context}"]
+= Archetypes
+
+{ProductFullName} includes *archetypes* that groups applications that are assessed for migration, according to common characteristics. The purpose of archetypes is to help you perform bulk assessments and reviews of applications.
+
```

Comment:
"...that group..." 

Actually, see my comments above on the "Archetypes" sub-section.